src/services/administration/iam/user/modules/user-management-modal/UserCreateModal.vue

```diff
@@ -86,7 +86,9 @@
                 <p-field-group :label="$t('IDENTITY.USER.FORM.ASSIGN_DOMAIN_ROLE')"
                                class="input-form"
                 >
-                    <p-select-dropdown v-model="formState.domainRole"
+                    <!-- CAUTION: Do not remove key binding at select dropdown. This is for initiating scroll parent to refresh fixed menu style. -->
```

Comment:
question: Does this apply to other pages, too?

Reply:
No, this page has tab in modal. That is why it is needed.